### DISCUSSION TITLE
SDXL Lightning

### DISCUSSION THREAD

**Discussion by Acly** [Acly/krita-ai-diffusion#433] [Research]
[SDXL Lightning](https://huggingface.co/ByteDance/SDXL-Lightning) is a new model for generating images with very low sample count (1-8). Unlike Turbo it uses the full SDXL resolution. It's similar to LCM, but is it better?

There are full checkpoint replacement models, including merges for popular models such as [DreamShaper](https://civitai.com/models/112902?modelVersionId=354657) and [Juggernaut](https://civitai.com/models/133005/juggernaut-xl). You can find instructions on how to setup sampling there if you want to try them.

But perhaps more interestingly, there are also LoRA which can be combined with any existing checkpoint. Here is how to setup the 8-step version:
1. Download the LoRA from here: https://huggingface.co/ByteDance/SDXL-Lightning/resolve/main/sdxl_lightning_8step_lora.safetensors
2. Put the LoRA in `ComfyUI/models/loras`
3. Go to Style settings. Add the LoRA.
4. Use the sampler for Lightning, 8 steps, CFG 1~2 _(requires [version 1.15.0!](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.15.0))_

![release-1 15 0-lightning](https://github.com/Acly/krita-ai-diffusion/assets/6485914/25ae6a9c-4ed8-433b-8814-8bfc8d7b79d9)

You can use it for both regular generation (Quality preset) and Live (Performance preset).

I'm particularly interested in how it performs in Live mode in comparison to LCM (the default), as it could potentially replace it. How well does it work at low strength (img2img)?

**Comment by ETCore7** [comment]
I don't notice a difference in speed in Live mode with SDXL. It's still too slow (for me, RTX3080) for practical use. With a 1.5 model there is no difference either, except the preview just copies what is drawn (default 30% strength). 

PS, Restore Defaults throws a Python error {ServerMode.undefined: -1), and after that the same error appears when trying to open the configuration screen again. All works after restart, though the server configuration then has been set to Online Service.
